### DISCUSSION TITLE
[Help] extract text by line

### DISCUSSION THREAD

**Discussion by supernovaMK** [jsvine/pdfplumber#1002] [Ask for help with specific PDFs]
In a situation where it is not recognized as a table, I would like to sequentially extract key values ​​for each column value based on the lines. I tried im.debug_tablefinder() but didnt work . and i usually used crop() but there are so many tables to crop
[example.pdf](https://github.com/jsvine/pdfplumber/files/12800475/example.pdf)
<img width="922" alt="image" src="https://github.com/jsvine/pdfplumber/assets/115217054/afc9b26f-034d-4a82-9239-3bc3312f3610">

example output)
key1
key2
column3:key3
column4:key4

**Comment by samkit-jain** [comment]
Hi @supernovaMK Appreciate your interest in the library. Specific to this PDF, you can use the following table extraction strategy,
```python
ts = {
    "vertical_strategy": "lines",
    "horizontal_strategy": "lines",
    "explicit_vertical_lines": [70, 520],
    "explicit_horizontal_lines": [80],
    "intersection_y_tolerance": 10,
}

page.extract_tables(ts)
```
which will give you the result as
![image](https://github.com/jsvine/pdfplumber/assets/15127115/a0593004-4f57-4fcf-ad96-c238bda5ba73)
```
['Key1', 'Key2']
['Column3\nKey3', 'Column4\nKey4']
['', '']
[None, '']
```

However, defining `explicit_horizontal_lines` like this to capture the start and end row may not be feasible for you. In that case, you can use the top and bottom of the middle vertical line for the same.
